@uportal/esco-content-menu-lit/README.md

```diff
@@ -0,0 +1,643 @@
+- `user-info-portlet-url`: type: `String`, default: `''`, an url/uri to the user information application,
+- `switch-org-portlet-url`: type: `String`, default: `''`, an optional url/url of a rest api to obtain institutional organization information,
+- `favorites-portlet-card-size`: type: possible value `auto|large|medium|small|smaller`, default: `small`, define the size of portlet-cards component into `favorite-content` component part,
+- `grid-portlet-card-size`: type: possible value `auto|large|medium|small|smaller`, default: `medium`, define the size of `portlet-cards` component into `grid-content` component part,
+- `hide-action-mode: type`: possible value `auto|always|never`, default: `auto`, define if we should show the actions, `auto` don't show on `small` breakpoint,
+- `user-org-id-attribute-name`: type: `String`, default: `'ESCOSIRENCourant[0]'`, the attribute object path to obtain the id of the organization to retrieve from the organization's api
+- `user-all-orgs-id-attribute-name`: type: `String`, default: `'ESCOSIREN`, the attribute object path to obtain all ids of the organizations linked to the user and to retrieve from the organization's api
+- `org-logo-url-attribute-name`: type: `String`, default: `'otherAttributes.ESCOStructureLogo[0]'`, the attribute object path to obtain the organization Picture from organization details obtained from the organization's api
+- `force-org-logo`: type: `String`, optional, an url/uri to provide an institutional picture overriding the default-org-logo and the institutional one obtained by an api (used to have an overview before updating change to every users from the api),
+- `show-favorites-in-slider`: type: `Boolean`, default: `false`, tell component to show favorites list in a slider.
+- `debug`: type: `Boolean`, default: `false`, for the demo/debug mode to be able to run in a standalone way (disable api call).
+
+#### Events
+
+##### Emitted
+
+- `toggle`: traget: `component`, details: `fname,chanId,isFavorite`, Emit when menu is open or close.
+
+#### Slots
+
+The HTML content of the component can also be modified using [slots](https://lit.dev/docs/components/shadow-dom/#slots).
+
+##### Menu Icon
+
+The `menu-icon` slot permit to apply a custom icon replacing the default Hamburger one. As example:
+
+```html
+<esco-hamburger-menu
+  default-org-logo="..."
+  api-url-org-info="..."
+  portlet-api-url="..."
+  layout-api-url="..."
+  organization-api-url="..."
+  user-info-api-url="..."
+  favorites-portlet-card-size="small"
+  grid-portlet-card-size="auto"
+  hide-action-mode="never"
+>
+  <div
+    slot="menu-icon"
+    style="background-image:url('https://mdbootstrap.com/img/svg/hamburger3.svg?color=FFF');height:20px;width:20px;"
+  ></div>
+</esco-hamburger-menu>
+```
+
+##### Menu Content
+
+The `menu-content` slot permit to apply an other content than the esco-content-menu sub-component.
```

Comment:
Are you meaning that developers can add other HTML tags in the `menu-content` slot?

Reply:
Yes it's customizable by slots